assets/helpers/user/user.js

```diff
@@ -22,9 +22,14 @@ const init = (dispatch: Function) => {
   const windowHasUser = window.guardian && window.guardian.user;
   const userAppearsLoggedIn = cookie.get('GU_U');
 
-  if (window.guardian && window.guardian.uatMode) {
+  if (window.guardian && window.guardian.uatMode != null) {
+    if (window.guardian.uatMode) {
```

Comment:
Is it possible to put this if condition in the wrapper if. i.e. 
`if(window.guardian && window.uatMode === true)`